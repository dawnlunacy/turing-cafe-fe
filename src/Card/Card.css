.card {
    background-color: #FFFFFF;
    border-radius: 10px;
    border: 5px solid #006767;
    color: #006767;
    display: inline-block;
    height: 250px;
    margin: 10px;
    width: 300px;
}

.cancel-btn {
    background-color: #008080;
    color: white;
    font-size: 1em;
    height: 35px;
    margin: 10px;
    width: 70%;

}