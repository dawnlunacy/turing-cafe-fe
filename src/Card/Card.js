import React from 'react';

const Card = () => {
    return (
        <p> Testing our communication from Card</p>
    )
}

export default Card;