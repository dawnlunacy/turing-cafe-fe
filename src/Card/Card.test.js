import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Card', () => {
  it('should return a card display the correct reservation informatino', () => {
    const wrapper = shallow(
      <Card />
    )
    expect(wrapper).toMatchSnapshot();
  })
})