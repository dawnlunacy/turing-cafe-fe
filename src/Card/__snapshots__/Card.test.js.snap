// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should return a card display the correct reservation informatino 1`] = `
<article
  className="card"
>
  <h3>
      
     
  </h3>
  <h4>
     
      
  </h4>
  <h4>
     
     
  </h4>
  <h4>
     
     
  </h4>
  <button
    className="cancel-btn"
  >
     Cancel 
  </button>
</article>
`;
